### ISSUE TITLE
Overridden errorImage(for:) and cvcImage(for:) are not called in subclass

### ISSUE THREAD

**Issue by mahavi** [stripe/stripe-ios#1848]
## Summary
Overridden class methods errorImage(for:) and cvcImage(for:) are not called in subclass. I guess, the similar issue as here https://github.com/stripe/stripe-ios/issues/1827

## Code to reproduce
```
import Stripe
import UIKit

final class PaymentCardTextField: STPPaymentCardTextField {
  override class func errorImage(for cardBrand: STPCardBrand) -> UIImage? { nil }
  override class func cvcImage(for cardBrand: STPCardBrand) -> UIImage? { nil }
}
```

## iOS version
13.x

## Installation method
SPM

## SDK version
21.7.0

**Comment by davidme-stripe** [comment]
Thanks for filing this, and sorry for the continued trouble with subclassing `STPPaymentCardTextField`! We'll add a fix to the next release.

**Comment by porter-stripe** [comment]
We fixed this in version 21.12.0.
